src/main/java/jp/ikedam/jenkins/plugins/updatesitesmanager/DescribedUpdateSite.java

```diff
@@ -181,15 +181,6 @@ public Descriptor getDescriptor()
     static public abstract class Descriptor extends hudson.model.Descriptor<DescribedUpdateSite>
     {
         /**
-         * Return the description of this DescribedUpdateSite.
-         * 
-         * Shown when selecting DescribedUpdateSite to create.
-         * 
-         * @return the description of this DescribedUpdateSite
-         */
-        abstract public String getDescription();

```

Comment:
No longer necessary as the "add" page was removed and now you select a update site to add from dropdown.